File: app/src/main/java/com/omar/retromp3recorder/app/recording/recorder/RecorderDefaults.java
package com.omar.retromp3recorder.app.recording.recorder;

public class RecorderDefaults {
    public final VoiceRecorder.BitRate bitRate;
    public final VoiceRecorder.SampleRate sampleRate;
    public final String filePath;

    public RecorderDefaults(VoiceRecorder.BitRate bitRate, VoiceRecorder.SampleRate sampleRate, String filePath) {
        this.bitRate = bitRate;
        this.sampleRate = sampleRate;
        this.filePath = filePath;
    }
}

File: app/src/main/java/com/omar/retromp3recorder/app/recording/recorder/RecorderDefaults.kt
package com.omar.retromp3recorder.app.recording.recorder

import com.omar.retromp3recorder.app.recording.recorder.Mp3VoiceRecorder.BitRate
import com.omar.retromp3recorder.app.recording.recorder.Mp3VoiceRecorder.SampleRate

data class RecorderDefaults(
    val bitRate: BitRate,
    val sampleRate: SampleRate,
    val filePath: String
)
